### ISSUE TITLE
Improve lore2sphinx buildbot results for `projects/core/howto/process.xhtml`

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4870]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#4870|
|Type|enhancement|
|Created|2011-02-11 05:30:53Z|
|Branch|https://github.com/twisted/twisted/tree/process-sphinx-4870|

like [#4566](https://github.com/twisted/twisted/issues/4566), except for the process howto

<details><summary>Searchable metadata</summary>

```
trac-id__4870 4870
type__enhancement enhancement
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__branches_process_sphinx_4870 branches/process-sphinx-4870
branch_author__screwtape__jdb screwtape, jdb
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1297402253000000 1297402253000000
changetime__1299985824000000 1299985824000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [30672]) Branching to 'process-sphinx-4870'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [30673]) Merge the process parts of the impossibly huge [[#4568](https://github.com/twisted/twisted/issues/4568)](https://github.com/twisted/twisted/issues/4568) branch

refs [#4870](https://github.com/twisted/twisted/issues/4870)
refs #4568

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="washort's avatar" src="https://avatars.githubusercontent.com/u/947487?s=50" width="50" height="50">](https://github.com/washort)<a name="note_5"></a>|@washort set status to `closed`|
|-|-|

(In [30972]) Merge process.xhtml changes from process-sphinx-4870.

Author: screwtape, jdb
Reviewer: washort
Fixes: [#4870](https://github.com/twisted/twisted/issues/4870)

Fixes formatting changes needed to make sphinx happy with process.xhtml.

**Comment by twisted-trac** [comment]
|[<img alt="washort's avatar" src="https://avatars.githubusercontent.com/u/947487?s=50" width="50" height="50">](https://github.com/washort)<a name="note_6"></a>|@washort commented|
|-|-|

Fixed some places where 'FD' was changed to 'fd'. Otherwise good; merged.
